.github/workflows/chaos.yml

```diff
@@ -40,30 +40,35 @@ jobs:
           kubectl get pods -n kube-system
           kubectl version
 
-      - name: Deploy Etcd Operator
+      - name: Deploy etcd with Helm
         run: |
-          git clone https://github.com/api7/etcd-operator.git --depth 1
-          bash etcd-operator/example/rbac/create_role.sh
-          kubectl create -f etcd-operator/example/deployment.yaml
-          bash ./t/chaos/setup_chaos_utils.sh ensure_pods_ready etcd-operator "True" 30
-          kubectl create -f etcd-operator/example/example-etcd-cluster.yaml
-          bash ./t/chaos/setup_chaos_utils.sh ensure_pods_ready etcd "True True True" 30
+          curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm install etcd bitnami/etcd --set replicaCount=3 --set auth.rbac.enabled=false
+          bash ./t/chaos/utils/setup_chaos_utils.sh ensure_pods_ready app.kubernetes.io/instance=etcd "True True True" 30
```

Comment:
You can just use `kubectl wait`.

Reply:
Wow it's cool! Thanks for telling me that!